internal/stubserver/stubserver.go

```diff
@@ -125,13 +128,15 @@ func (ss *StubServer) StartClient(dopts ...grpc.DialOption) error {
 
 	cc, err := grpc.Dial(ss.Target, opts...)
 	if err != nil {
+		cc.Close()
```

Comment:
When `grpc.Dial()` returns a non-nil error, it is expected to return a `nil` ClientConn. So, this call to `Close()` in that case will `panic`.

Reply:
Right, fixed.